admin/class-yoast-dashboard-widget.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class to change or add WordPress dashboard widgets
+ */
+class Yoast_Dashboard_Widget {
+
+	/**
+	 * @var WPSEO_Statistics
+	 */
+	protected $statistics;
+
+	/**
+	 * @param WPSEO_Statistics $statistics The statistics class to retrieve statistics from.
+	 */
+	public function __construct( WPSEO_Statistics $statistics = null ) {
+		if ( null === $statistics ) {
+			$statistics = new WPSEO_Statistics();
+		}
+
+		$this->statistics = $statistics;
+
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_dashboard_stylesheet' ) );
+		add_action( 'wp_dashboard_setup', array( $this, 'add_dashboard_widget' ) );
+	}
+
+	/**
+	 * Adds dashboard widget to WordPress
+	 */
+	public function add_dashboard_widget() {
+		wp_add_dashboard_widget(
+			'wpseo-dashboard-overview',
+			__( 'Yoast SEO Posts Overview', 'wordpress-seo' ),
+			array( $this, 'display_dashboard_widget' )
+		);
+	}
+
+	/**
+	 * Display the dashboard widget
+	 */
+	public function display_dashboard_widget() {
+		$statistics = $this->statistic_items();
+
+		include WPSEO_PATH . '/admin/views/dashboard-widget.php';
+	}
+
+	/**
+	 * An array representing items to be added to the At a Glance dashboard widget
+	 *
+	 * @return array
+	 */
+	private function statistic_items() {
```

Comment:
Can you put this method underneath the public methods. We try to keep a sequence like: public, protected, private